app/workers/approve_stolen_listing_worker.rb

```diff
@@ -2,6 +2,47 @@ class ApproveStolenListingWorker < ApplicationWorker
   sidekiq_options queue: "notify"
 
   def perform(bike_id)
-    StolenTwitterbotIntegration.new.send_tweet(bike_id)
+    bike = Bike.find(bike_id)
+    new_tweet = TwitterTweeterIntegration.new(bike).create_tweet
```

Comment:
I think it might make sense to put this into a background job, and have that call the email (or put them both into the same job).

But that can be done later.

Reply:
> I think it might make sense to put this into a background job

Why's that? 🤔 (`TwitterTweeterIntegration` is invoked in a bg job)

Reply:
Omg I have no idea how I got lost here. Thought this was a controller. Ignore